README.md

```diff
@@ -214,6 +207,17 @@ To see gemini working end-to-end, check out [the HackMD example](examples/hackmd
 * Be sure to test out both the backup and restore process to ensure Gemini is working properly
 * VolumeSnapshots simply grab the current state of the volume, without respect for things like in-flight database transactions. You may find you need to stop the application in order to get a consistently usable VolumeSnapshot.
 
+<!-- Begin boilerplate -->
+## Join the Fairwinds Open Source Community
+
+The goal of the Fairwinds Community is to exchange ideas, influence the open source roadmap,
+and network with fellow Kubernetes users.
+[Chat with us on Slack](https://join.slack.com/t/fairwindscommunity/shared_invite/zt-e3c6vj4l-3lIH6dvKqzWII5fSSFDi1g)
+[join the user group](https://www.fairwinds.com/open-source-software-user-group) to get involved!
```

Comment:
Did you intentionally remove the 'or' between these two items?

Reply:
...I did not 😞 

Good catch!

Reply:
Will fix in a followup